cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -76,6 +76,10 @@ spec:
                 fieldPath: metadata.namespace
           - name: LEADER_ELECTION
             value: "{{ .Values.leaderElection }}"
+        {{- range $key, $value := .Values.extraEnvironmentVars }}
```

Comment:
@BigSammich I wonder if we should also allow for env vars to be set on RBAC manager for consistency (though I know it isn't relevant for your use case). I think we should at least call these `extraEnvironmentVarsCrossplane` for consistency with other inputs, and consider adding `extraEnvironmentVarsRBACManager`. (Feel free to shorten to `extraEnvVars____` if you like also)

Reply:
Agree with the consistency. Changes made